cpp/log/cert.cc

```diff
@@ -583,8 +583,8 @@ util::Status Cert::OctetStringExtensionData(int extension_nid,
   // Callers don't care whether extension is missing or invalid as they
   // usually call this method after confirming it to be present.
   const StatusOr<void*> ext_struct = ExtensionStructure(extension_nid);
-  if (!ext_struct.ok()
-      && ext_struct.status().CanonicalCode() == Code::NOT_FOUND) {
+  if (!ext_struct.ok() &&
```

Comment:
Don't think we had an opinion on this...
It's always been:
1) Google C++ Style Guide (which, AFAIR doesn't talk about exactly where to break)
2) then tie-break by falling back to what clang-format does.